### PR TITLE
docs(bodiless-psh): Update CDN doc

### PR DESCRIPTION
**Pull request by hvanyo**
## Changes
Updating documentation for Fastly CDN

### CODE REVIEW

File: packages/bodiless-psh/README.md [hunk 1]
```diff
@@ -596,6 +596,49 @@ In order to avoid redirect chains, pay attention on destination path protocol an
...
+    ```
+1. Verify or Update your `.platform.app.yaml` expiration time for your files.
+    ```
+    web:
+        locations:
+            '/':
+                expires: 6h  
+    ```
+
+Once completed, the master env deployed on Platform.sh should be on Fastly CDN.  You may have to fine tuen the expires for your static resources and set certain ones to longer to leverage browser caching.
```
Comment (beliayeu) [inline packages/bodiless-psh/README.md:624]: tuen -> tune

File: packages/bodiless-psh/README.md [hunk 2]
```diff
@@ -596,6 +596,49 @@ In order to avoid redirect chains, pay attention on destination path protocol an
 The trailing slash should be appended to the configure item if platform environment adds trailing slash to url by default.
 see [Platform.sh Documentation Redirects](https://docs.platform.sh/configuration/routes/redirects.html)
 
+## Using Fastly CDN
+
+Platform.sh integrates with Fastly via EZ platform for Fastly.  
+
+1. Step one is to obtain your Fastly Service ID & Key from Fastly.
```
Comment (malachye) [inline packages/bodiless-psh/README.md:603]: Remove "step one is to". Capitalize 'O' in 'obtain'

File: packages/bodiless-psh/README.md [hunk 3]
```diff
@@ -596,6 +596,49 @@ In order to avoid redirect chains, pay attention on destination path protocol an
...
+
+Platform.sh References:
+* [Set Fastly Credentials on Platform.sh](https://docs.platform.sh/frameworks/ez/fastly.html#set-credentials-on-platformsh)
+* [HTTP Cache](https://docs.platform.sh/configuration/routes/cache.html)
+* [Router Cache](https://docs.platform.sh/languages/php/tuning.html#ensure-that-the-router-cache-is-properly-configured)
+* [Expires](https://docs.platform.sh/configuration/app/web.html#locations)
+* [How to Guide: How to configure caching for static assets](https://community.platform.sh/t/how-to-configure-caching-for-static-assets/187)
+
+
+If there are issues or need to troubleshoot, here are some good resources
```
Comment (malachye) [inline packages/bodiless-psh/README.md:634]: 1. insert "you" 
"If there are issues or **_you_** need to troubleshoot..."

2. add a colon (:) after "resources"
"here are some good resources:"

File: packages/bodiless-psh/README.md (hunk 2 above)
Comment (hvanyo) [reply packages/bodiless-psh/README.md]: Fixing

File: packages/bodiless-psh/README.md (hunk 3 above)
Comment (hvanyo) [reply packages/bodiless-psh/README.md]: fixed

File: packages/bodiless-psh/README.md (hunk 1 above)
Comment (hvanyo) [reply packages/bodiless-psh/README.md]: Fixed

**Pull request merged**